docs/installation.md

```diff
@@ -7,12 +7,13 @@ and quay.io for the public registry. We also test on minikube version v0.35+.
 * [OKD](https://www.okd.io/)
   * Experimental: 3.11
```

Comment:
This will also be need to be updated or removed altogether as the current master wont work..

Reply:
Updated. Also modified the intro paragraph which discusses minikube/shift/etc versions